include/openssl/bio.h

```diff
@@ -145,6 +145,16 @@ extern "C" {
 
 # define BIO_CTRL_DGRAM_SET_PEEK_MODE      71
 
+/* internal BIO see include/internal/bio.h:
+ * # define BIO_CTRL_SET_KTLS                 72
+ * # define BIO_CTRL_SET_KTLS_CTRL_MSG        74
+ * # define BIO_CTRL_CLEAR_KTLS_CTRL_MSG      75
+ */
+
+#  define BIO_CTRL_GET_KTLS                 73
+#  define BIO_get_ktls_send(b)         \
+     BIO_ctrl(b, BIO_CTRL_GET_KTLS, 0, NULL)
```

Comment:
rename BIO_CTRL_GET_KTLS as BIO_get_ktls_send ?

Reply:
It must be capital case as all other defines.

I'll adjust the name to make the direction more clear in the define.